app/konnect/dev-portal/customization/netlify.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Host your Dev Portal with Netlify
+content_type: how-to
+---
+
+You can use the open-source Dev Portal to display your APIs to developers on a self-hosted website.
+
+This guide shows you how to configure an example open-source Dev Portal with Netlify. Using this example as a starting point, you can adjust the code of the sample app and self-host it on a frontend application of your choice.
+
+The example application is pre-configured with the following:
+* All [features in the {{site.konnect_short_name}}-hosted Dev Portal](/konnect/dev-portal/customization/)
+* Localization is set to English by default. You can configure it using [`src/locales`](https://github.com/Kong/konnect-portal/tree/main/src/locales).
+* [UI test automation through Cypress](https://github.com/Kong/konnect-portal/tree/main/cypress)
+
+## Prerequisites
+
+* [A {{site.konnect_saas}} account](/konnect/getting-started/access-account/)
+* [Yarn 1.22.x installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [A Netlify account](https://www.netlify.com/)
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Custom Self-Hosted UI Domain** field. For example, `client.mycompany.com`.
+1. Add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
+    ```sh
+    api.mycompany.com   CNAME   3600    client.mycompany.com
```

Comment:
These are wrong.  The CNAME needs to point the Custom Hosted Domain to the Konnect Portal Domain.  

more like 

```
api.mycompany.com   CNAME   3600    portal29asdfj278.us.portal.konghq.com
```